src/runners/RunnerUtil.ml

```diff
@@ -156,20 +158,23 @@ let parse_cli () =
         ),
       "Path(s) to libraries separated with ':' (';' on windows)");
     ("-init", Arg.String (fun x -> r_init_file := Some x), "Path to initialization json");
-    ("-cf", Arg.Unit (fun () -> r_cf := true), "Run cashflow checker and print results.");
+    ("-cf", Arg.Unit (fun () -> r_cf := true), "Run cashflow checker and print results");
+    ("-cf-token-field", Arg.String (fun s -> r_cf_token_fields := s :: !r_cf_token_fields), "Make the cashflow checker consider a field to be money (only allowed if -cf is set)");
```

Comment:
Optional: An alternate way of doing this would be, if `-cf-token-field` is specified, then `-cf` is assumed.

Reply:
That's probably better. Fixed.